src/components/ui/navbar.tsx

```diff
@@ -38,7 +38,7 @@ const Navbar = ({
     <>
       <div
         className={cn(
-          " hidden w-full items-center justify-between py-7 md:flex",
+          " hidden w-full items-center justify-between py-7 md:flex backdrop-filter backdrop-blur-lg bg-opacity-30",
```

Comment:
This doesn't work at all because the navbar is not fixed; the blur effect cannot be seen.